src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -120,10 +128,29 @@ void function_call_harness_generatort::implt::generate(
   // create body for the function
   code_blockt function_body{};
 
+  generate_nondet_globals(function_body);
   setup_parameters_and_call_entry_function(function_body);
   add_harness_function_to_goto_model(std::move(function_body));
 }
 
+void function_call_harness_generatort::implt::generate_nondet_globals(
+  code_blockt &function_body)
+{
+  if(nondet_globals)
+  {
+    for(const auto &symbol_table_entry : *symbol_table)
+    {
+      const auto &symbol = symbol_table_entry.second;
+      if(
+        symbol.is_static_lifetime && symbol.is_lvalue &&
```

Comment:
We _think_ this means "global", but we weren't entirely sure. This is basically reverse engineered from a `--show-symbol-table`.

Reply:
Looks good to me.